data/panel.css

```diff
@@ -229,7 +230,6 @@ input[type=range]::-moz-focus-outer {
 
 .video-progress {
     background: #ffffffaa;
-    border-radius: 20px;
```

Comment:
why did you remove border-radius here and below in `.video-progress[value]::-moz-progress-bar`?